torchbenchmark/models/llama_v2_7b_8h/__init__.py

```diff
@@ -0,0 +1,15 @@
+from torchbenchmark.tasks import NLP
+from torchbenchmark.util.framework.huggingface.model_factory import HuggingFaceModel, HuggingFaceAuthMixin
+
+class Model(HuggingFaceModel, HuggingFaceAuthMixin):
+    task = NLP.LANGUAGE_MODELING
+    DEFAULT_TRAIN_BSIZE = 1
+    DEFAULT_EVAL_BSIZE = 1
+    DEEPCOPY = False 
+
+    def __init__(self, test, device, batch_size=None, extra_args=[]):
+        HuggingFaceAuthMixin.__init__(self)
+        super().__init__(name="llama_v2_7b_8h", test=test, device=device, batch_size=batch_size, extra_args=extra_args)
+
+    def train(self):
+        return NotImplementedError("7b LLAMA model will OOM on CI GPU machines")
```

Comment:
Currently our criteria to accept model is that the eager mode test does not OOM on CI GPU machines (A10G and A100, respectively). We cannot guarantee that they won't OOM on the optim userbenchmark.